SETUP.md

```diff
@@ -38,6 +38,21 @@ nvm alias default 8
 brew install yarn
 ```
 
+### Install Rust
```

Comment:
Do we still need this in the monorepo?

Reply:
Good point. I thought this is a good place for general engineering setup as well, and I see how it might be confusing. Suggestion for anywhere else instead?

Reply:
Good point. I thought this is a good place for general engineering setup as well, and I see how it might be confusing. Suggestion for anywhere else instead?